xlators/mgmt/glusterd/src/glusterd-geo-rep.c

```diff
@@ -79,12 +79,10 @@ static char *gsync_no_restart_opts[] = {"checkpoint", "log_rsync_performance",
 void
 set_gsyncd_inet6_arg(runner_t *runner)
 {
-    xlator_t *this = NULL;
     char *af;
     int ret;
 
-    this = THIS;
-    ret = dict_get_str(this->options, "transport.address-family", &af);
```

Comment:
Half of the functions in this file are static, that call one each other. We could have reduced the number of calls to THIS by passing the result between functions. We can save it for a future iteration of course.

Reply:
Ack. I'll add this in the issue so that it can be tracked..